Pod/Classes/AUMediaPlayer.m

```diff
@@ -279,7 +279,11 @@ - (void)updatePlayerWithItem:(id<AUMediaItem>)item error:(NSError * __autoreleas
     [self prepareForCurrentItemReplacementWithItem:item];
     
     NSURL *url = nil;
-    if ([_library itemIsDownloaded:item]) {
+    if([item isLocalItem]) {
+        url = [NSURL fileURLWithPath: [item localPath]];
+        NSLog(@"Playback will occur with a completely local item at: %@", url); 
```

Comment:
No, ofc not but it followed from the other statements in that sequence which logs in a similar way?